airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -347,6 +347,7 @@ def __init__(
         client_type: Optional[str] = None,
         resource_type: Optional[str] = None,
         config: Optional[Config] = None,
+        **kwargs,
```

Comment:
If we don't _do_ anything with these kwargs why do we need to accept them here?

Reply:
That was being used to swallow the unused args when the Operators were creating the hook in their constructors.  I will remove it when I correct those.